src/sentry/snuba/ourlogs.py

```diff
@@ -0,0 +1,190 @@
+import logging
+
+import sentry_sdk
+from sentry_protos.snuba.v1.endpoint_trace_item_table_pb2 import Column as RPCColumn
+from sentry_protos.snuba.v1.endpoint_trace_item_table_pb2 import TraceItemTableRequest
+from sentry_protos.snuba.v1.trace_item_attribute_pb2 import AttributeAggregation, AttributeKey
+from snuba_sdk import Column, Condition
+
+from sentry.search.eap.columns import ResolvedColumn, ResolvedFunction
+from sentry.search.eap.ourlog_columns import OURLOG_DEFINITIONS
+from sentry.search.eap.resolver import SearchResolver
+from sentry.search.eap.types import CONFIDENCES, ConfidenceData, EAPResponse, SearchResolverConfig
+from sentry.search.events.fields import get_function_alias
+from sentry.search.events.types import EventsMeta, EventsResponse, SnubaData, SnubaParams
+from sentry.snuba.dataset import Dataset
+from sentry.snuba.metrics.extraction import MetricSpecType
+from sentry.snuba.query_sources import QuerySource
+from sentry.utils import snuba_rpc
+from sentry.utils.snuba import process_value
+
+logger = logging.getLogger("sentry.snuba.ourlogs")
+
+
+def categorize_column(column: ResolvedColumn | ResolvedFunction) -> RPCColumn:
+    if isinstance(column, ResolvedFunction):
+        return RPCColumn(aggregation=column.proto_definition, label=column.public_alias)
+    else:
+        return RPCColumn(key=column.proto_definition, label=column.public_alias)
```

Comment:
This is identical to the `spans_rpc` one right? 🤔 maybe we should move this to a classmethod on the resolver?